src/pages/Strategies/components/StrategyCard/StrategyCard.tsx

```diff
@@ -1,58 +1,56 @@
-import { ReactNode } from 'react';
 import { useTranslation } from 'react-i18next';
+import { Link } from 'react-router-dom';
 
 import { formatPercentage } from '@/common/utils/utils';
 import { Card, CardProps, CoinIcon, Flex, H1, P, Strong } from '@/component-library';
-import { StrategyData, StrategyRisk } from '@/types/strategies';
+import { STRATEGIES, StrategyType } from '@/types/strategies';
+import { PAGES } from '@/utils/constants/links';
 
+import { useGetStrategyInsights } from '../../hooks/use-get-strategy-insights';
 import { StrategyTag } from '../StrategyTag';
 import { StyledEarningCard, StyledEarnSection } from './StrategyCard.style';
 
 type Props = {
-  title: ReactNode;
-  description: ReactNode;
-} & StrategyData;
+  strategyType: StrategyType;
+};
 
 type InheritAttrs = Omit<CardProps, keyof Props>;
 
 type StrategyCardProps = Props & InheritAttrs;
 
-const StrategyCard = ({
-  interestType,
-  interestPercentage,
-  currency,
-  description,
-  risk,
-  title,
-  ...props
-}: StrategyCardProps): JSX.Element => {
+const StrategyCard = ({ strategyType, ...props }: StrategyCardProps): JSX.Element => {
   const { t } = useTranslation();
+  const { interest } = useGetStrategyInsights(strategyType);
+  const { descriptionCard, title, currency, risk, path, tags } = STRATEGIES[strategyType];
 
-  const interestTypeLabel = interestType === 'apr' ? t('apr') : t('apy');
-  const interestPercentageLable = formatPercentage(interestPercentage.toNumber());
+  const interestPercentageLable = formatPercentage(interest);
 
   return (
-    <Card {...props} alignItems='center' gap='spacing4'>
-      <Flex alignSelf='flex-start' gap='spacing1'>
-        <StrategyTag variant='risk' risk={risk} />
-        {(risk === StrategyRisk.LOW || risk === StrategyRisk.MEDIUM) && <StrategyTag variant='passive-income' />}
-      </Flex>
-      <CoinIcon size='xl2' ticker={currency.ticker} />
-      <H1 weight='bold' size='base' align='center' rows={1}>
-        {title}
-      </H1>
-      <StyledEarningCard justifyContent='center'>
-        <StyledEarnSection size='xs' align='center'>
-          Earn up to
-          <Strong size='base'>
-            {interestPercentageLable} {interestTypeLabel}
-          </Strong>
-        </StyledEarnSection>
-      </StyledEarningCard>
-      <P color='tertiary' size='xs' align='center'>
-        {description}
-      </P>
-    </Card>
+    <Link to={`${PAGES.STRATEGIES}/${path}`}>
```

Comment:
This correct semantic is an html error. An anchor cannot have div's as child. Html errors will affect our page score.

Reply:
You are right. However this issue is not so straight-forward. I think we should route with link, your approach of programatical redirect in onClick handler did not work well because it was reloading the whole page (beating the purpose of SPA). Another problem with not routing using anchors is that the page is less accessible and it's not possible to open link on new page using CTRL+click.  Maybe we should place a button there?

Reply:
We can still use the Link but we will need to change the children semantics. Which is possible. I will take a look